benchmark/elastic/issue_elastic_queries.py

```diff
@@ -0,0 +1,126 @@
+    elif queryname == "search id.orig_h + sum orig_bytes":
+        query = "curl -X GET \"localhost:9200/test/_search?format=json&pretty\" -H 'Content-Type: application/json' -d'{\"query\" : {\"term\" : { \"id.orig_h\" : \""+ arguments[0] +"\" }}, \"aggs\": {\"orig_bytes\": {\"sum\": {\"field\": \"orig_bytes\"}}}}'"
+    elif queryname == "search id.orig_h + count by schema":
+        query = "curl -X GET \"localhost:9200/test/_search?format=json&pretty\" -H 'Content-Type: application/json' -d'{\"query\" : {\"term\" : { \"id.orig_h\" : \""+ arguments[0] +"\" }}, \"aggs\": {\"schema\": {\"terms\": {\"field\": \"_path\"}}}}'"
+    else:
+        query= ""
+    return query
+
+def issueQueries():
+    real_list = []
+    query_name_list = []
+    argument_list = []
+    validation_list =[]
+    
+    reset()
+
+    with open(input_directory + workload_file ) as f:
+        data = ndjson.load(f)
+        for d in data:
+            if (d['query'] in queries):
+                queryname = d['query']
+                query = getQuery(queryname, d['arguments'])
+                if query:
+                    if queryname in aggregation_fields:
+                        prepForAggregation(aggregation_fields[queryname])
+
+                    queryoutput = runQuery(query)
+                    real_list.append(queryoutput["executiontime"])
+                    query_name_list.append(d['query'])
+                    argument_list.append(d['arguments'])
+                    validation_list.append(queryoutput["hits"])
+
+                    if queryname in aggregation_fields:
+                        cleanFromAggregation(aggregation_fields[queryname])
+                else:
+                    print('Query Not Supported: ' + str(d['query']))
+            else:
+                print('Query Not Supported: ' + str(d['query']))
+     	
+    num_queries = len(real_list)
+    output_df = pd.DataFrame({'index': range(num_queries)})
+    output_df = output_df.set_index('index')
+
+    output_df.insert(0, 'system', ['Elastic'] * num_queries)
+    output_df.insert(1, 'in_format', ['index'] * num_queries)
+    output_df.insert(2,'out_format', ['elastic'] * num_queries)
+    output_df.insert(3, 'query', query_name_list)
+    output_df.insert(4,'start_time', [0] * num_queries)
+    output_df.insert(5,'real', real_list)
+    output_df.insert(6, 'user', [np.nan] * num_queries)
+    output_df.insert(7, 'sys',[np.nan] * num_queries)
+    output_df.insert(8, 'argument_0', argument_list)
+    output_df.insert(9, 'validation', validation_list)
+
+    output_df.to_csv(output_directory + output_file, na_rep='NaN')
+    return output_df
+
+def clearcache():
+    clearcache = "curl -X POST \"localhost:9200/test/_cache/clear?pretty\""
+    os.system(clearcache)
+
+def restartElastic():
+    os.system("sudo systemctl stop elasticsearch.service")
+    os.system("sudo systemctl start elasticsearch.service")
+    os.system("sleep 30")
+
+def reset():
+    clearcache()
```

Comment:
If you're going to restart Elastic Search anyway, does clearing the cache beforehand have any effect? I would have expected that restarting elastic would yield an empty cache, but my intuition could be wrong here.